invokeai/frontend/src/features/parameters/store/postprocessingSlice.ts

```diff
@@ -40,6 +41,16 @@ export const postprocessingSlice = createSlice({
   name: 'postprocessing',
   initialState,
   reducers: {
+    setAllPostProcessingParameters: (
+      state,
+      action: PayloadAction<InvokeAI.Metadata>
+    ) => {
+      const { type, hires_fix } = action.payload.image;
+
+      if (type === 'txt2img' && typeof hires_fix === 'boolean')
+        state.hiresFix = hires_fix;
+      // Strength of img2img used in hires_fix is not currently exposed in the Metadata for the final image.
```

Comment:
This doesn't handle the case where a image did not use hires, and hires is already enabled. Let's instead cast the value of `hires_fix` to bool to cover all situations:

```suggestion typescript
      if (type === 'txt2img') {
        // Strength of img2img used in hires_fix is not currently exposed in the Metadata for the final image.
        state.hiresFix = Boolean(hires_fix);
      }
```

Reply:
> This doesn't handle the case where a image did not use hires 

The UI always has this field, even when false, but it seems the CLI doesn't handle this quite the same. Good catch -- will change. 

Other [similar assignments](https://github.com/invoke-ai/InvokeAI/blob/main/invokeai/frontend/src/features/parameters/store/generationSlice.ts#L290) may need to be changed in the future (outside of the scope here).

Reply:
I just saw this and added the (tiny bit of) code needed for the CLI to write `hires_fix` to the metadata. Let me know if more is needed.